### PR TITLE
Dynamically swap themes for asciinema

### PR DESCRIPTION
**Pull request by Elsie19**

### CODE REVIEW

**Comment by Elsie19** [comment]
@ajstrongdev I can't assign you but I am pinging you.

File: client/src/pages/Home.tsx [hunk 1]
```diff
@@ -98,12 +100,20 @@ const Home: FC = () => {
                     <Heading size={'lg'} mb='3' mt='10'>
                         {t('home.showcase.title')}
                     </Heading>
-                    <AsciinemaFrame autoplay loop id='538264' />
+                    <AsciinemaFrame
+                        autoplay
+                        loop
+                        id='538264'
+                        dark={colorMode === 'light' ? true : false}
+                    />
```
Comment (ajstrongdev) [inline client/src/pages/Home.tsx:108]:
```
<AsciinemaFrame
    autoplay
    loop
    id='538264'
    dark={colorMode === 'light'}
/>
```

File: client/src/pages/Home.tsx [hunk 2]
```diff
@@ -98,12 +100,20 @@ const Home: FC = () => {
...
 
                     <Heading size={'lg'} mb='3'>
                         {t('home.showcase.packageSearch')}
                     </Heading>
-                    <AsciinemaFrame id='538265' />
+                    <AsciinemaFrame
+                        id='538265'
+                        dark={colorMode === 'light' ? true : false}
+                    />
```
Comment (ajstrongdev) [inline client/src/pages/Home.tsx:116]:
```
<AsciinemaFrame
    id='538265'
    dark={colorMode === 'light'}
/>
```

**Review by ajstrongdev — CHANGES_REQUESTED** [review]
Not tested but I believe this is the correct solution

**Review by ajstrongdev — APPROVED** [review]
LGTM

**Pull request merged**